backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -128,6 +153,41 @@ private native static void setImageData (ArrayBufferView pixels, int width, int
 	private ImageElement imageElement;
 	private VideoElement videoElement;
 
+	public boolean ready = false;
+
+	/** Creates a new Pixmap instance from the given encoded image data.
+	 *
+	 * @param encodedData the encoded image data
+	 * @param offset the offset
+	 * @param len the length */
+	public Pixmap (byte[] encodedData, int offset, int len) {
+		ready = false;
```

Comment:
Is this meant to be assigned `true` here like it is in the non-emulated Pixmap? I don't think `ready` is used at all, in either Pixmap impl, and should probably be removed unless the JS specifically needs it.

Reply:
I am setting it as false here so the dev can check if this is loaded before instantiating a new texture using this Pixmap, otherwise it fails.

When `pixmap.ready` you can do `new Texture(pixmap)`.